worker/worker.js

```diff
@@ -0,0 +1,59 @@
+// Set up Firebase.
+var firebase = require('firebase');
+var config = {
+  apiKey: "AIzaSyAzPSc-CtNCs0zbU0KkIu5NzcRFnhkeabo",
+  authDomain: "gridproject-fd25f.firebaseapp.com",
+  databaseURL: "https://gridproject-fd25f.firebaseio.com",
+  storageBucket: "gridproject-fd25f.appspot.com",
+  messagingSenderId: "187144675114"
+};
+firebase.initializeApp(config);
+firebase.auth().signInAnonymously().catch(function(error) {
+    console.log("Could not auth", error);
+});
+
+// The frequency in seconds in which to make some move on the grid.
+var FREQUENCY_S = 5;
+
+function changeColor() {
+
+	var dimensions = {};
+
+	// connect to grid and count rows
+	var rowCountRef = firebase.database().ref('grid0/');
+	rowCountRef.once("value", function(rowSnapshot) {
+		dimensions.rows = rowSnapshot.numChildren();
```

Comment:
This won't work because we converted the db to only having the values that have been added. If you look at it now it only has grid0/r0/c0 and that's it. You might have to just manually specify the dimensions you want to cover.